src/window.ts

```diff
@@ -258,9 +258,8 @@ export class ShellWindow {
                 wm_class = this.name(ext)
             }
 
-            return !this.meta.is_skip_taskbar()
```

Comment:
@mmstick do you know what this line was supposed to do?

Reply:
That I don't recall.